keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -414,48 +416,64 @@ Both these cases will be handled properly, here's an example:
 type Object struct {
      // This field has an invalid zero-value, empty string is not a
      // valid string.
-     // +default=default-name
-     Name string
+     // +default="default-name"
+     Name string `json:"name,omitempty"`
```

Comment:
this is no example for the cases above:
1. `""` is valid
2. the default is not the zero value.

Reply:
How do you know "" is valid?

Reply:
Why does it matter whether it is valid or not? Why not the other cases? This is not obvious.